src/components/Header/Header.js

```diff
@@ -5,16 +5,16 @@ import './Header.css'
 const Header = () => {
   return (
     <div class="navigation-container flex-hz">
-    <div class="navigation-item-1">
+      <div class="navigation-item-1">
         <h5 class="lib-name-1">Keep <span class="lib-name-2">App</span></h5>
-    </div>
-    <i class="fa fa-bars"></i>
-  
-    <div class="navigation-item-3 flex-hz">
+      </div>
+      <i class="fa fa-bars"></i>
+
+      <div class="navigation-item-3 flex-hz">
         <Link to="/signin"><button class="btn btn-primary">Login</button></Link>
-       
+
+      </div>
     </div>
-</div>
   )
 }
 
```

Comment:
One small suggestion is that you could take care of these spacings and justifications when you create the component for the first time to maintain consistency.
Otherwise, when you raise a PR for a feature, it might show as a change in these files which can be misleading.